ballerina/xmldata.bal

```diff
@@ -54,27 +56,29 @@ public isolated function fromJson(json jsonValue, JsonOptions options = {}) retu
             }
             json value = jMap.toArray()[0];
             if value is json[] {
-                return getElement("root", check traverseNode(value, {}, options, jMap.keys()[0]),
-                                check getAttributesMap(value, options = options));
+                _ = check getNamespacesMap(value, namespaces, {}, options);
```

Comment:
I think we can move this to line number 58, Seems like `getNamespaceMap` doesn't depend on if condition